python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -314,6 +318,10 @@ def fit(self, data, validation_data=None, epochs=1, batch_size=32, validation_mo
                 warnings.warn("Xshards is collected to local since the "
                               "forecaster is non-distribued.")
                 data = xshard_to_np(data)
+            if isinstance(validation_data, SparkXShards) and not self.distributed:
+                warnings.warn("Xshards is collected to local since the "
+                              "forecaster is non-distribued.")
+                validation_data = xshard_to_np(validation_data)
```

Comment:
okay, this reminds me that we have not support validation data in distributed forecaster, we may add one line to tell our users that the validation data will be ignored under distributed mode

Reply:
done